GRAPHICS/DoesLatestBriefStringExist.md

```diff
@@ -0,0 +1,33 @@
+---
+ns: GRAPHICS
+---
+## _0x5E657EF1099EDD65
+
+```c
+// 0x5E657EF1099EDD65 0x91A081A1
+BOOL DOES_LATEST_BRIEF_STRING_EXIST(int briefType);
+```
+
+```
+Checks if brief text is available.
+
+Brief types:
+0 - Dialogue Brief
+1 - Help Text Brief
+2 - Mission Objective Brief
+```
+
+```c
```

Comment:
This example wouldn't really be applicable for others, and doesn't provide much use for what its actually used for

Reply:
ah, can i somehow edit this pull request or do i have to make a new one?